unittests/solid_3D_ele/4C_solid_3D_ele_calc_lib_test.cpp

```diff
@@ -234,4 +239,106 @@ namespace
 
     FOUR_C_EXPECT_NEAR(x_centroid, x_centroid_ref, 1e-15);
   }
+
+  namespace
+  {
+    template <Core::FE::CellType celltype>
+    Discret::Elements::ElementNodes<celltype> get_element_nodes()
+    {
+      std::srand(0);
```

Comment:
Why not write out the arbitrary numbers instead of hiding them behind random? Would make it easier to debug or test elsewhere.